Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreateTest.py

```diff
@@ -60,10 +60,6 @@ def test_exec(self):
             ComponentList='bank1', ComponentMaxTranslation=0.2, ComponentMaxRotation=10)
         # Check source position
         row = mtd['cal_adjustments'].row(0)
-        assert_allclose([row[name] for name in ('Xposition', 'Yposition', 'Zposition')], [0., 0., -10.0], atol=0.001)
-        # Check position of first bank
-        row = mtd['cal_adjustments'].row(1)
-        target_position, target_orientation, target_rotation = [5.18, -0.32,  0.20], [0.001, 0.999, -0.027], 98.0
```

Comment:
You just left `target_position, target_orientation, target_rotation` undefined. The following lines won't "compile"